addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -2518,6 +2518,7 @@ const Wysiwyg = Widget.extend({
         }
     },
     resetEditor: function (value, options) {
+        this.$editable[0].removeEventListener('focus', this._joinPeerToPeer);
```

Comment:
This was done here: https://github.com/odoo/odoo/pull/111454/commits/0001c1f09c1fd52935136a7fcde6bd7745d8f75c